specification/network/resource-manager/Microsoft.Network/stable/2024-10-01/examples/AllVirtualNetworkGatewayRadiusServerSecretsList.json

```diff
@@ -0,0 +1,24 @@
+{
+  "parameters": {
+    "api-version": "2024-10-01",
+    "subscriptionId": "subid",
```

Comment:
>subid [](http://example.com/codeflow?start=23&length=5)

nit, valid guid looking subscription ids make for better examples

Reply:
Fixed.